discovery-infra/tests/conftest.py

```diff
@@ -69,19 +83,23 @@ def get_api_client(offline_token=env_variables['offline_token'], **kwargs):
     return assisted_service_api.create_client(url, offline_token, **kwargs)
 
 
+@pytest.fixture(scope="session")
+def is_dev_env():
```

Comment:
why is it a fixture?

Reply:
using it from base_test.py

Reply:
i think we have something relevant to this in conftest, can u check?

Reply:
Why not just a function?

Reply:
cause than ill need to import conftest into base, and it is not the intention of it

Reply:
What's the problem with that?

Reply:
Fixed